src/thorlabs_cube/driver/tcube/tpz.py

```diff
@@ -33,46 +37,38 @@ async def handle_message(self, msg):
                 )
             )
 
-    async def set_position_control_mode(self, control_mode):
+    async def set_position_control_mode(self, control_mode: int) -> None:
         """Set the control loop mode.
 
         When in closed-loop mode, position is maintained by a feedback signal
         from the piezo actuator. This is only possible when using actuators
         equipped with position sensing.
 
         :param control_mode: 0x01 for Open Loop (no feedback).
-
             0x02 for Closed Loop (feedback employed).
-
             0x03 for Open Loop Smooth.
-
             0x04 for Closed Loop Smooth.
         """
         await self.send(
-            Message(MGMSG.PZ_SET_POSCONTROLMODE, param1=1, param2=control_mode)
+            Message(MGMSG.PZ_SET_POSCONTROLMODE, param1=_CHANNEL, param2=control_mode)
         )
 
-    async def get_position_control_mode(self):
+    async def get_position_control_mode(self) -> int:
         """Get the control loop mode.
 
         :return: Returns the control mode.
-
             0x01 for Open Loop (no feedback).
-
             0x02 for Closed Loop (feedback employed).
-
             0x03 for Open Loop Smooth.
-
             0x04 for Closed Loop Smooth.
-
         :rtype: int
         """
         get_msg = await self.send_request(
-            MGMSG.PZ_REQ_POSCONTROLMODE, [MGMSG.PZ_GET_POSCONTROLMODE], 1
+            MGMSG.PZ_REQ_POSCONTROLMODE, [MGMSG.PZ_GET_POSCONTROLMODE], _CHANNEL
```

Comment:
```suggestion
            MGMSG.PZ_REQ_POSCONTROLMODE, [MGMSG.PZ_GET_POSCONTROLMODE], Tpz._CHANNEL
```
Use class constants.